src/directory/directory.mjs

```diff
@@ -330,7 +330,7 @@ export const directory = {
           {
             title: 'SMS flows',
             route: '/lib/auth/sms_flows',
-            filters: ['flutter', 'ios']
+            filters: ['flutter', 'ios', 'android']
```

Comment:
You need the one above this too (MFA)

Reply:
The SMS flows page requires code snippets. Do you want to add that in a separate PR or this one?

Reply:
done